README.md

```diff
@@ -61,7 +61,12 @@ Sign up for [IBM Watson Studio](https://dataplatform.ibm.com).
 * Click on **Notebook**.
 * Click on **From URL**.
 * Fill in a name for the notebook (e.g. "diabetes-prediction").
-* Copy and paste this URL into the notebook URL field:  https://raw.github.com/IBM/summit-health-ml/master/diabetes-prediction.ipynb
+* Copy and paste this URL into the notebook URL field:
+
+```
```

Comment:
This formatting prevents it from being a clickable link. We usually don't format links as code.